source/_includes/common-tasks/backups.md

```diff
@@ -33,19 +33,22 @@ Before creating a backup, check if you can reduce the size of the backup. This i
 The automatic backup process creates a backup on a predefined schedule and also deletes old, redundant backups.
 
 1. Go to {% my supervisor_backups title="**Settings** > **System** > **Backups**" %}.
-2. Under **Automatic backups**, select **Configure automatic backups**.
-3. Enable automatic backup.
-4. Define the backup schedule. It is recommended to back up daily.
+2. Under **Set up backups**, select **Set up backups**.
+3. To be able to restore encrypted backups, download the emergency kit and store it somewhere safe.
+   - To learn more about backup encryption, refer to the documentation on the [backup emergency kit](/more-info/backup-emergency-kit/).
```

Comment:
`To be able to restore encrypted backups` sounds like something optional

Reply:
good point, thx